.github/workflows/ci.yml

```diff
@@ -44,7 +34,7 @@ jobs:
 
       - uses: actions-rs/cargo@v1
         with:
-          command: build
+          command: check
```

Comment:
Build would do codegen and link-checking, `check` does not (but we're no longer linking dynamic libs, so probably fine to not really test that?).